src/components/Menu/Menu.jsx

```
@@ -1,11 +1,9 @@
-import React, { Component, PropTypes } from 'react';
+import React, { Component } from 'react';
 
-import Message from '../Items/Items';
+import Items from '../Items/Items';
 
 export default class Menu extends Component {
-  constructor(props) {
-    super(props);
-  }
+
```

the constructor was removed to help suppress errors, ignore this one